.eslintrc

```diff
@@ -41,6 +41,7 @@
       }
     ],
     "no-console": 0,
+    "no-undef": 0,
```

Comment:
I guess this is a problem, but before we change this eslint. I think we would need to bring this rule up with the others in the team (y)

Reply:
This is an issue with typescript; it complains when using certain primitive types, mainly Record.